hw/xbox/xid.c

```diff
@@ -158,15 +160,14 @@ static const USBDesc desc_xbox_gamepad = {
 static const XIDDesc desc_xid_xbox_gamepad = {
     .bLength = 0x10,
     .bDescriptorType = USB_DT_XID,
-    .bcdXid = 1,
+    .bcdXid = 0x100,
```

Comment:
What is this field?

By the way, I noticed these fields have their type width prefixed (like Microsoft coding style), but this field is incorrectly prefixed as it's actually a 16-bit field. We should probably drop the prefixes (I don't think we don't really do that in QEMU code, that I've seen), or change the `b` to `w`.